ApiBundle/Transformer/ContentCollectionTransformer.php

```diff
@@ -30,7 +31,7 @@ public function transform($contentCollection, $contentType = null)
             array()
         ));
 
-        if ($contentType) {
+        if ($contentType && $this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_CREATE_CONTENT_TYPE_FOR_CONTENT)) {
```

Comment:
@itkg-ngilain  i check the correct right, this is the same right used for the NEW ACTION in contentController. The right for create content type is: ROLE_ACCESS_CREATE_CONTENT_TYPE. you can verify this in the contentTypeController.